src/frontend/js/index.tsx

```diff
@@ -60,8 +60,11 @@ document.addEventListener('DOMContentLoaded', event => {
         .join('');
       // Sanity check: only attempt to access and render components for which we do have a valid name
       if (isComponentName(componentName)) {
-        // Determine a lang (localeCode) based on the `data-locale` attribute
-        const locale = element.getAttribute('data-locale') || 'en';
+        // Determine an ISO15897 locale from a BCP47/RFC5646 locale (eg. en-US to en_US)
+        const locale = (element.getAttribute('data-locale') || 'en-US').replace(
```

Comment:
In Django languages are set to "en-us" and not "en-US" so maybe you should still force capitalize on the last 2 characters?

Reply:
See here: https://github.com/django/django/blob/master/django/conf/global_settings.py#L51

Reply:
This is done by `rfc_5646_locale`. We already need the capitalization to be done for the `<html lang>` because otherwise it's not a valid BCP47/RFC5646 locale.